buildSrc/src/main/java/com/uber/okbuck/core/util/ReplaceUtil.java

```diff
@@ -18,18 +18,14 @@
 
     @SuppressWarnings("ResultOfMethodCallIgnored")
     static void copyResourceToProject(String resource, File destination, Map<String, String> templates) {
-        try {
-            InputStream inputStream = FileUtil.class.getResourceAsStream(resource);
-            destination.getParentFile().mkdirs();
-            InputStreamReader reader = new InputStreamReader(inputStream);
-            TemplateReader replacingReader = new TemplateReader(reader, new TemplateMapResolver(templates));
-
-            OutputStream outputStream = new FileOutputStream(destination);
+        destination.getParentFile().mkdirs();
```

Comment:
Same thing here `new FileOutputStream(destination)` depends on this file having been created.